src/EditorFeatures/Core/Wrapping/BinaryExpression/BinaryExpressionCodeActionComputer.cs

```diff
@@ -37,12 +38,12 @@ private class BinaryExpressionCodeActionComputer :
                 _exprsAndOperators = exprsAndOperators;
                 _preference = options.GetOption(CodeStyleOptions.OperatorPlacementWhenWrapping);
 
-                var generator = SyntaxGenerator.GetGenerator(document);
-                var indentationString = OriginalSourceText.GetOffset(binaryExpression.Span.Start)
-                                                          .CreateIndentationString(UseTabs, TabSize);
+                _indentationService = service.GetIndentationService();
```

Comment:
we have a Document, so if IBlankLineIndentationService is a language service, we can just get it from there using `.GetLanguageService<...>()`.

Reply:
This pattern is carried over from before. I'll file an issue to address separately if it's not fixed before merge.

Reply:
➡️ Moved to #32832